### DISCUSSION TITLE
PrismaClient returning undefined - Fixed

### DISCUSSION THREAD

**Discussion by MarioWork** [prisma/prisma#13898] [Q&A] [answered]
Hello there, I have a project where I am using the Web Framework Fastify with Prisma and my `PrismaClient` is always returning `undefined`. I have been trying to fix this problem but cannot seem to know how, the connection works fine but the `PrismaClient`is not working. 
I started the project with `npm init fastify`

Thank you in advance.

[Github Repo for testing](https://github.com/MarioWork/mcu-api/tree/mario/feat/generate-api-key-route)

Fixed

**Comment by austincrim** [comment, marked as answer]
Good to hear! If you think your solution would be helpful for other users, feel free to share it 🚀.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
